lib/human_player.rb

```diff
@@ -0,0 +1,114 @@
+=begin
+Winning patterns
+  #horizontal wins
+  [0][0], [0][1], [0][2]
+  [1][0], [1][1], [1][2]
+  [2][0], [2][1], [2][2]
+
+  #vertical wins
+  [0][0], [1][0], [2][0]
+  [0][1], [1][1], [2][1]
+  [0][2], [1][2], [2][2]
+
+  #diagonal wins
+  [0][0], [1][1], [2][2]
+  [0][2], [1][1], [2][0]
+=end
+
+class Board
+    def initialize
+      @boxNumbers = [
+        ["1", "2", "3"],
+        ["4", "5", "6"],
+        ["7", "8", "9"]
+      ]
+      @top_row = "| | | |"
+      @mid_row = "| | | |"
+      @bot_row = "| | | |"
+  
+      @rules = ["|1|2|3|", "|4|5|6|", "|7|8|9|"]
+    end
+  
+    def show
+      puts @top_row
+      puts @mid_row
+      puts @bot_row
+    end
+  
+    def show_rules
+      puts "Welcome to Tic Tac Toe!"
+      puts "These are the diffenrent move positions you can make:"
+      puts @rules, ""
```

Comment:
I love how modular and reusable the code are for `#show_rules` and `#show`, but since we are practicing TDD, I'd like to see tests for these print statements captured somewhere.